expcore/gui/prototype.lua

```diff
@@ -348,5 +350,12 @@ Gui._prototype_element.on_text_changed = event_handler_factory(defines.events.on
 -- @tparam function handler the event handler which will be called
 Gui._prototype_element.on_value_changed = event_handler_factory(defines.events.on_gui_value_changed)
 
+--- Element Events.
+-- @section customEvents
+Gui.events = {
```

Comment:
Event table is already defined and you are overwriting it. See [ln85](https://github.com/Cooldude2606/scenario/blob/bcde7943add47ffea8c0894b5b6e462f238282c0/expcore/gui/prototype.lua#L85)

Reply:
https://github.com/Cooldude2606/scenario/pull/3/commits/2d90d1dcdd3071c7820a0083f006e4f5b5e61dde